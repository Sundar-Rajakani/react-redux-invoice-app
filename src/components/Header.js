import React from 'react';
import { NavLink } from 'react-router-dom';
const Header = () => (
  <header>
    <h1>Invoice Management</h1>
    <NavLink to="/" activeClassName="isActive" exact={true}>Home</NavLink>
    <NavLink to="/add" activeClassName="isActive">AddInvoice</NavLink>
    <NavLink to="/help" activeClassName="isActive">Help</NavLink>
  </header>
);

export default Header;
