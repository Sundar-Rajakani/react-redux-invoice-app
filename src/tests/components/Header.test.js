import React from 'react';
// import ReactShallowRenderer from 'react-test-renderer/shallow';
import { shallow } from 'enzyme';
import Header from '../../components/Header';


test('should render header correctly', () => {
  // const renderer = new ReactShallowRenderer();
  // expect(renderer.getRenderOutput()).toMatchSnapshot();
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});