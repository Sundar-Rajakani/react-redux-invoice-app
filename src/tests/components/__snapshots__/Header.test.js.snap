// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<header>
  <h1>
    Invoice Management
  </h1>
  <NavLink
    activeClassName="isActive"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="isActive"
    ariaCurrent="true"
    to="/add"
  >
    AddInvoice
  </NavLink>
  <NavLink
    activeClassName="isActive"
    ariaCurrent="true"
    to="/help"
  >
    Help
  </NavLink>
</header>
`;
